nvidia.ps1

```diff
@@ -136,7 +136,7 @@ elseif ($archiverProgram -eq "$env:programfiles\WinRAR\WinRAR.exe") {
 
 # Installing drivers
 Write-Host "Installing Nvidia drivers now..."
-$install_args = "-s -noreboot -noeula"
+$install_args = "-passive -noreboot -noeula -s"
```

Comment:
What does passive do?

Reply:
I was deploying it [that way ](https://lazyadmin.nl/it/deploy-nvidia-drivers/ )when during my sccm times.  according to setup.cfg its preventing any user input but leaves a progressbar up. 
```
"s" property="SilentInstall"
"prestage" property="PrestageInstall"
"i" property="ReducedUI"
"forcereboot" property="RebootRequired"
"noreboot" property="IgnoreReboot"
"k" property="RebootRequired"
"n" property="IgnoreReboot"
"passive" property="ProgressOnly"
"noeula" property="SkipEula"
"nofinish" property="SkipFinish"
```
IIRC why we were using it -> in case the installer fails no additional windows would pop up that require input from a user, it fails and end the script. 

And now that I think about it: take it with a grain of salt I rely on things coworkers were claiming as it's mostly undocumented and only based on their experience.